File: javascript/gl.js
const fs = require("fs")

function render(to_render) {
    to_render = to_render.replace("0", " ")
    to_render = to_render.replace("1", "#")
    return to_render
}

function render_to_file(to_render, file) {
    to_render = to_render.replace("0", " ")
    to_render = to_render.replace("1", "#")
    fs.writeFile(file, to_render, (err) => {
        if (err) throw err;
    }
    )
}

File: python/gl.py
def render(string: str):
    string = string.strip().replace("0", " ").replace("1", "#")
    return string

def render_to_file(file, string: str):
    string = string.strip().replace("0", " ").replace("1", "#")
    with open(file, "a") as f:
        f.write(string)
        f.close()
        return string